### DISCUSSION TITLE
How do you get rid of the property error in VS Code?

### DISCUSSION THREAD

**Discussion by splashsky** [Annoraaq/grid-engine#155] [Q&A] [answered]
Naturally, with the way Phaser registers plugins for scenes, VS Code shows a red line and errors anywhere I use `this.gridEngine` within a scene. Is there an easy fix for this issue so I can clean up my interface?

Thanks! :smile:

**Comment by Annoraaq** [comment, marked as answer]
Is this related with this issue you experienced before? https://github.com/Annoraaq/grid-engine/discussions/47

I guess it is a typescript project? In that case it should suffice to add 
```private gridEngine: GridEngine;```
to your Scene class.

**Reply by splashsky** [reply]
Ah yes. I apologize - I forgot I had this before, haha. Gotta search before posting, indeed 😛
